docs/installation/basic.md

```diff
@@ -70,7 +70,11 @@ bower install
 * **Step 4** - Create application environment variables.
 
 ```sh
-export DATABASE_URL=postgresql://open_event_user:test@127.0.0.1:5432/test
+create .env file inside open-event-orga-server folder
+# Enter the following and save:
```

Comment:
@magdalenesuo instead of this, add a `.env.example` file to the repo root. And mention it here. That's how most projects usually do. The user will be familiar with it and he/she can easily do a `cp .env.example .env`